app/src/main/res/layout/fragment_main.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/colorPrimary"
+        android:paddingTop="?attr/actionBarSize">
+
+        <ImageView
+            android:id="@+id/ivLogo"
+            android:layout_marginTop="32dp"
```

Comment:
It would be nice if you'll try to organize your resources in the app on the early stages. There is a practice to avoid usage of "dp" values directly in layout but moves them to resources (e.g. dimensions) or even organize styles hierarchy.